ModPatches/Rimsenal Federation/Patches/Rimsenal Federation/ThingDefs_Misc/Fed_MechWeapons.xml

```diff
@@ -20,6 +20,7 @@
 			<soundCast>RS_ShotHarmonizer</soundCast>
 			<muzzleFlashScale>17</muzzleFlashScale>
 			<soundCastTail>GunTail_Heavy</soundCastTail>
+			<ignorePartialLoSBlocker>true</ignorePartialLoSBlocker>
```

Comment:
While we're at it, we probably want to let some of these weapons (including this one) target locations--which it doesn't _appear_ they can currently do.

Reply:
Yeah I'll go ahead and check it out.